rskj-core/src/test/java/co/rsk/peg/federation/builders/P2shErpFederationBuilder.java

```diff
@@ -0,0 +1,121 @@
+package co.rsk.peg.federation.builders;
```

Comment:
Not sure I'd put it in a `builders` package. It creates circular dependency between `federation` and `builders`